docs/content/latest/secure/tls-encryption/connect-to-cluster.md

```diff
@@ -48,12 +40,12 @@ The `ysqlsh` CLI is available in the `bin` directory of your YugabyteDB home dir
 
 To connect to a remote YugabyteDB cluster, you need to have a local copy of `ysqlsh` available. You can use the `ysqlsh` CLI available on a locally installed YugabyteDB.
 
-To open the local `ysqlsh` CLI and access your YugabyteDB cluster, run `ysqlsh` with the following flags set:
+To open the local `ysqlsh` CLI and access your YugabyteDB cluster, run `ysqlsh` with the following flags defined:
 
 - host: `-h <node-ip-address>` (required for remote node; default is `127.0.0.1`)
 - port: `-p <port>` (optional; default is `5433`)
 - user: `-U <username>` (optional; default is `yugabyte`)
-- TLS/SSL: `"sslmode=require"` (this flag is required)
+- TLS/SSL: `"sslmode=require"` (required)
```

Comment:
minor: there is some useful information we could add here about sslmode, which could be relevant to most users. From https://www.postgresql.org/docs/9.1/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS, there are two modes in particular that are useful - `verify-ca` and `verify-full`. We could consider explaining them here or linking to the postgres docs if @m-iancu thinks that is okay.